docs/core/diagnostics/dotnet-trace.md

```diff
@@ -180,6 +188,42 @@ To collect traces using `dotnet-trace`:
 
 - Stop collection by pressing the `<Enter>` key. `dotnet-trace` will finish logging events to the *trace.nettrace* file.
 
+## Launch a child application and collect a trace from its startup using dotnet-trace
+
+NOTE: This works for apps running .NET 5.0 or later only.
+
+Sometimes it may be useful to collect a trace of a process from its startup. For apps running .NET 5.0 or later, it is possible to do this by using dotnet-trace.
+
+Suppose there is a .NET 5.0 application named `hello.exe` with `arg1` and `arg2` that you want to collect a trace from its startup.
+
+To do this, run the following command:
+
+```console
+dotnet-trace collect -- hello.exe arg1 arg2
+```
+
+This will launch `hello.exe` with `arg1` and `arg2` as its command line arguments and collect a trace from its runtime startup:
```

Comment:
People would probably figure it out but I suggest adding similar text to what the previous examples used to make it unambiguous that this is the output of the command, not the command itself.

 "The preceding command generates output similar to the following:"